charybdis/disasm.py

```diff
@@ -14,6 +14,282 @@ def decode_insn(rom: bytes, index: int) -> Optional[DecodedInsn]:
     decoded = None
     byte = rom[index]
     size = 1
-    if byte == 0x00:
-        decoded = insn.Insn(name=insn.InsnName.NOP)
+    match byte:
+        ###########
+        # LD r, r #
+        ###########
+
+        # LD B, r
+        case 0b01_000_000:
```

Comment:
Let's use hex constants instead of binary. It's much easier to tell if a hex sequence is continuous at a glance.